ide-support/revdocsparser.livecodescript

```diff
@@ -2169,10 +2169,10 @@ private function escape pString, pConvertLineEndings
    replace quote with ("\" & quote) in pString
    
    if pConvertLineEndings is true then
-      replace (numToChar(13) & CR) with (CR & numToChar(13)) in pString
-      replace (CR & numToChar(13)) with CR in pString
+      replace CRLF with "\n" in pString
+      replace numToCodepoint(13) with "\n" in pString
    end if
-   replace CR with "\n" in pString
+   replace LF with "\n" in pString
```

Comment:
@bwmilby same issue as I noted on the IDE patch there's a problem with `pConvertLineEndings` false here potentially not escaping carriage return

Reply:
@montegoulding all uses in this file omit the second parameter which means that it will be false (27).  Many times it is an actual literal like "doc" or a variable that holds an array key.
since text is converted upon import to LF and we are not seeing bugs in the docs related to this, I don't think it is an issue.
Having said that, adding an else would make it "complete" in case a carriage return did make it in did arise.